rocAL/rocAL/include/api/rocal_api_meta_data.h

```diff
@@ -60,8 +60,13 @@ extern "C" RocalMetaData ROCAL_API_CALL rocalCreateTFReaderDetection(RocalContex
 ///
 /// \param rocal_context
 /// \param source_path path to the coco json file
+/// \param mask enable polygon masks
```

Comment:
is_output desciption is missing

Reply:
Currently is_output param for all reader functions are unused so its not been added as param anywhere. I'll check with Shobana and confirm once before adding the param desc